extensions/sql-migration/src/dialog/skuRecommendationResults/GenerateProvisioningScriptDialog.ts

```diff
@@ -0,0 +1,223 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import path = require('path');
+import { MigrationStateModel } from '../../models/stateMachine';
+import * as constants from '../../constants/strings';
+import * as styles from '../../constants/styles';
+import * as utils from '../../api/utils';
+import { logError, TelemetryViews } from '../../telemetry';
+import { IconPathHelper } from '../../constants/iconPathHelper';
+import { SelectStorageAccountDialog } from './SelectStorageAccountDialog';
+
+export class GenerateProvisioningScriptDialog {
+
+	private static readonly CloseButtonText: string = 'Close';
+
+	private dialog: azdata.window.Dialog | undefined;
+	private _isOpen: boolean = false;
+
+	private _disposables: vscode.Disposable[] = [];
+	private _armTemplateTextBox!: azdata.TextComponent;
+	private _armTemplateText!: string;
+
+	constructor(public model: MigrationStateModel, public _targetType: utils.MigrationTargetType) { }
+
+	private async initializeDialog(dialog: azdata.window.Dialog): Promise<void> {
+		return new Promise<void>((resolve, reject) => {
+			dialog.registerContent(async (view) => {
+				try {
+					const flex = this.CreateSaveArmTemplateContainer(view);
+
+					this._disposables.push(view.onClosed(e => {
+						this._disposables.forEach(
+							d => { try { d.dispose(); } catch { } });
+					}));
+
+					await view.initializeModel(flex);
+					resolve();
+				} catch (ex) {
+					reject(ex);
+				}
+			});
+		});
+	}
+
+	private CreateSaveArmTemplateContainer(_view: azdata.ModelView): azdata.FlexContainer {
+
+		const armTemplateDescription = _view.modelBuilder.text().withProps({
+			value: constants.TARGET_PROVISIONING_DESCRIPTION,
+			CSSStyles: {
+				...styles.BODY_CSS,
+				'margin-bottom': '8px',
+				'height': '36'
+			}
+		}).component();
+
+		this._armTemplateTextBox = _view.modelBuilder.text().withProps({
+			value: this._armTemplateText,
+			width: '100%',
+			height: '100%',
+			CSSStyles: {
+				'font': '12px "Monaco", "Menlo", "Consolas", "Droid Sans Mono", "Inconsolata", "Courier New", monospace',
```

Comment:
no need to add font css. It will be same across ADS

Reply:
done.